### DISCUSSION TITLE
Creat WP fail!

### DISCUSSION THREAD

**Discussion by sumax0913** [EasyEngine/easyengine#1587] [Support Requests] [answered]
I've successfully installed ee and wp on GCP Ubutu18 VM a lot of times. But It's seem went wrong when I try to do it again just yesterday. 

The procedure to install ee is normal. Then installing WP, I use  `sudo ee site create sub.example.com --type=wp` First time I get the following :

`Error: Could not find apache2-utils installed in services_global-nginx-proxy_1.`

Then the second try I get this:

```
Warning: Could not create user sub.example.com-8GyUwg. Please check logs.
Warning: Initiating clean-up.
Success: Site sub.example.com deleted.
Report bugs here: https://github.com/EasyEngine/site-type-wp
```
Anyone can help? Thanks!

**Comment by mayank1059** [comment, marked as answer]
You are not alone I am facing a similar issue tried multiple times

**Reply by pradyukrish** [reply]
Found this on https://github.com/EasyEngine/site-type-wp/issues/200 :

Comment https://github.com/EasyEngine/site-type-wp/issues/200#issuecomment-932902076 by @bravenguyendotcom:
>If I reload/restart the nginx-proxy service, the error disappear:
>ee service reload nginx-proxy
>But another error pops up:
>could not create user abc.com-xxxxx
>Then I add parameter --local-db, everything is just fine.

Try using the --local-db option. This option will create a separate container for your site's db instead of using the global db container.

**Reply by sumax0913** [reply]
Great~ It's work!
Thank @pradyukrish & Thank @mayank1059 
Thanks
